google/cloud/bigtable/async_instance_data.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGTABLE_ASYNC_INSTANCE_DATA_H_
```

Comment:
Why not simply: `cloud/bigtable/instance_list.h` for the file name?

Reply:
`*_data.h` is a bit too generic.  What do you think of `async_instance_list_responses.h`?  A bit long, but easier to understand what should be there.